neofetch

```diff
@@ -1578,8 +1579,11 @@ getbattery () {
                         done
                         return
                     fi
-
-                    battery="${batteries[0]}%"
+						if [ $power_status = "Charging" ]; then
```

Comment:
Should be `if [ "$power_status" == "Charging" ]; then`.

Also, fix the indentation here.